packages/date-picker/test/keyboard-input.test.js

```diff
@@ -791,14 +775,6 @@ describe('keyboard', () => {
           datePicker._focusAndSelect();
         });
 
-        it('should change after validate on Esc with clear button', async () => {
```

Comment:
**note:** The clear button logic doesn't depend on autoOpenDisabled, so a single general test is enough: `should validate before change event on clear button click`.